api/src/main/java/com/fit_goal/Notification.java

```diff
@@ -0,0 +1,21 @@
+package com.fit_goal;
+
+import com.fit_goal.enums.Message;
+import com.fit_goal.enums.Subject;
+import lombok.Getter;
+import lombok.Setter;
+
+
+
```

Comment:
Please work with code style every time, no need to keep empty rows.

Reply:
refactored